core-web/libs/portlets/edit-ema/portlet/src/lib/store/dot-uve.store.ts

```diff
@@ -1,30 +1,31 @@
 import { patchState, signalStore, withComputed, withMethods, withState } from '@ngrx/signals';
 
-import { computed } from '@angular/core';
+import { computed, untracked } from '@angular/core';
 
 import { withEditor } from './features/editor/withEditor';
 import { withFlags } from './features/flags/withFlags';
 import { withLayout } from './features/layout/withLayout';
 import { withLoad } from './features/load/withLoad';
 import { ShellProps, TranslateProps, UVEState } from './models';
 
-import { DotPageApiResponse } from '../services/dot-page-api.service';
+import { DotPageApiParams, DotPageApiResponse } from '../services/dot-page-api.service';
 import { UVE_FEATURE_FLAGS } from '../shared/consts';
 import { UVE_STATUS } from '../shared/enums';
 import { getErrorPayload, getRequestHostName, sanitizeURL } from '../utils';
 
 const initialState: UVEState = {
     isEnterprise: false,
     languages: [],
-    pageAPIResponse: null,
+    pageAPIResponse: null, // This should be called `pageAsset`
     currentUser: null,
     experiment: null,
     errorCode: null,
-    params: null,
+    pageParams: null,
+    viewParams: null, // This should be called `viewSettings`/`editorViewSettings`?
     status: UVE_STATUS.LOADING,
-    isTraditionalPage: true,
-    canEditPage: false,
-    pageIsLocked: true
+    isTraditionalPage: true, // This should be a computed property
+    canEditPage: false, // This should be a computed property
+    pageIsLocked: true // This should be a computed property
```

Comment:
I believe I tried that but It didn't work because we need this in other parts of the store and you cannot access computed properties there, just the state

Reply:
I think NgRx was updated recently in the repo, is worth giving a try to see if they somehow support that now

Reply:
I'll do the tests, then I give it a shot!